src/components/keymap/header/keymap-header.component.ts

```diff
@@ -14,10 +14,9 @@ import { KeymapActions } from '../../../store/actions';
 })
 export class KeymapHeaderComponent {
     @Input() keymap: Keymap;
+    @Output() deleted: EventEmitter<boolean> = new EventEmitter();
```

Comment:
What is the purpose of emitting a boolean here? As I see its emitted value is irrelevant, because the only place which is listening doesn't care about the value.